### ISSUE TITLE
Gulp fails

### ISSUE THREAD

**Issue by bkr32** [StartPolymer/polymer-starter-kit-plus#45]
Hi, firstly thank you for expanding PSK and making this kit open source its really amazing,

i have downloaded it and followed all instructions listed in documentation however when it comes to using gulp or gulp serve:dist , i get the following error

`Ignoring duplicate element definition: undefined
Ignoring duplicate element definition: undefined
ERROR finding /Users/ahmed bakir/Documents/32_site/dist/themes/default-theme/fonts/fonts.css
[20:32:23] Plumber found unhandled error:
 Error in plugin 'gulp-vulcanize'
Message:
    ENOENT: no such file or directory, open 'C:\Users\ahmed bakir\Documents\32_site\dist\themes\default-theme\fonts\fonts.css'
Details:
    errno: -4058
    code: ENOENT
    syscall: open
    path: C:\Users\ahmed bakir\Documents\32_site\dist\themes\default-theme\fonts\fonts.css
    fileName: C:\Users\ahmed bakir\Documents\32_site\dist\elements\base-bundle.html`

i hope you can assist me thanks

**Comment by JosefJezek** [comment]
You need run `gulp init`. Please send me feedback.

**Comment by bkr32** [comment]
i ran it several times, i instead put /fonts/fonts.css into the default-themes dir, where fonts.css is head.css

**Comment by JosefJezek** [comment]
Please send me output from `gulp init`...

**Comment by bkr32** [comment]
`gulp init
fatal: No names found, cannot describe anything.
Warning: Can't run "git describe" for determine app version
[09:27:45] Using gulpfile ~\Documents\32_site\gulpfile.js
[09:27:45] Starting 'init'...
[09:27:45] Starting 'download:analytics'...
[09:27:50] Starting 'download:fonts'...
[09:27:53] Finished 'download:fonts' after 2.65 s
[gulp] Downloading https://www.google-analytics.com/analytics.js... Done
[09:27:54] Copy downloaded analytics.js file to app/scripts dir: all files 26.3 kB
[09:27:54] Finished 'download:analytics' after 9.9 s
[09:27:54] Finished 'init' after 9.9 s`


full output

**Comment by jaikme** [comment]
Same problem here

**Comment by JosefJezek** [comment]
Do you have installed git? Try git clone this repo... then run gulp init in repo.

**Comment by jaikme** [comment]
yes. I'm using Cmder MINGW64 with Git Bash.
I was trying to deploy to the Firebase, so I cloned the repo again, and started fresh. Installed npm and bower dependencies, ran the gulp init. OK. Now I tryed run $ gulp (Build & Vulcanize) again. Same error.
Vulcanize task is trying to find dist\themes\default-theme\fonts\fonts.css file
This file is not being generated by the task 'fonts', because task 'download:fonts' is not downloading any font from the font.list file, because **config.theme argument is undefined at download-fonts.js task file...**
so i figure out that the corrent param is the "**_config.appTheme_**" not "config.theme" at download-fonts.js  file.

Now is all working.

[gulp.txt](https://github.com/StartPolymer/polymer-starter-kit-plus/files/287968/gulp.txt)
